### PR TITLE
testgrid: print out all failed tests for visibility.

### PR DESCRIPTION
**Pull request by siyuanfoundation**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

https://github.com/etcd-io/etcd/issues/17773

### CODE REVIEW

**Comment by siyuanfoundation** [comment]
cc @serathius

File: scripts/measure-testgrid-flakiness.sh [hunk 1]
```diff
@@ -10,6 +10,12 @@ then
 fi
 
 pushd ./tools/testgrid-analysis
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-e2e-amd64
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-unit-test-amd64
+# ci-etcd-e2e-amd64 and ci-etcd-unit-test-amd64 runs 6 times a day. Keeping a rolling window of 14 days.
+go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-e2e-amd64 --max-runs=84
```
Comment (serathius) [inline scripts/measure-testgrid-flakiness.sh:14]: Could we instead make it time based and not run based?

File: scripts/measure-testgrid-flakiness.sh [hunk 2]
```diff
@@ -10,6 +10,12 @@ then
 fi
 
 pushd ./tools/testgrid-analysis
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-e2e-amd64
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-unit-test-amd64
+# ci-etcd-e2e-amd64 and ci-etcd-unit-test-amd64 runs 6 times a day. Keeping a rolling window of 14 days.
```
Comment (serathius) [inline scripts/measure-testgrid-flakiness.sh:13]: Could we instead make it time based and not run based?

**Review by serathius — APPROVED** [review]

File: scripts/measure-testgrid-flakiness.sh (hunk 2 above)
Comment (siyuanfoundation) [reply scripts/measure-testgrid-flakiness.sh:13]: done

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @siyuanfoundation

**Review by serathius — APPROVED** [review]

**Review by ahrtr — APPROVED** [review]

**Pull request merged**